build.gradle

```diff
@@ -123,21 +123,25 @@ subprojects { project ->
   checkerFramework {
     checkers = [
              'com.amazon.checkerframework.compliance.kms.ComplianceChecker',
-             'com.amazon.checkerframework.cryptopolicy.CryptoPolicyComplianceChecker'
+             'com.amazon.checkerframework.cryptopolicy.CryptoPolicyComplianceChecker',
+             'org.checkerframework.checker.https.HTTPSChecker'
     ]
-    extraJavacArgs = ['-Xmaxerrs', '1000', '-AsuppressWarnings=cast.unsafe', "-Astubs=$rootProject.projectDir/stubs"]
+    extraJavacArgs = ['-AprintVerboseGenerics', '-Xmaxerrs', '1000', '-AsuppressWarnings=cast.unsafe', "-Astubs=$rootProject.projectDir/stubs"]
```

Comment:
We only added that option for debugging. There's no reason to keep it, since it slows down the code.